.gitignore

```diff
@@ -0,0 +1,6 @@
+.DS_Store
```

Comment:
In fact I added the `.gitignore` because showed me `node_modules` as new directory. I don't have a `~/.gitignore` set up and, although it makes sense, really makes it harder for less experienced people to contribute. People who know git will easily configure it the way they want, but those who don't know how to do it will have additional challenges.